### PR TITLE
Add README and package.json

### PR DESCRIPTION
**Pull request by dnewh**
Add initial README and package.json files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "transact-sdk-javascript",
+  "version": "0.0.0-alpha",
```
Comment (peterschwarz) [inline package.json]: This should probably have the initial version.  `0.1.0`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Transact SDK Javascript
+
+
+License
+-------
+Hyperledger Sawtooth software is licensed under the [Apache License Version 2.0](LICENSE) software license.
```
Comment (peterschwarz) [inline README.md]: Sawtooth -> Transact

File: package.json (hunk 1 above)
Comment (dnewh) [reply package.json]: updated

File: README.md (hunk 2 above)
Comment (dnewh) [reply README.md]: updated

**Pull request merged**